lib/measures/tbd/measure.rb

```diff
@@ -130,6 +138,76 @@ def run(model, runner, user_arguments)
       end
     end
 
+    if gen_kiva
+      # Although one may choose to auto-generate Kiva settings and objects,
+      # there must be at least one valid foundation-facing floor in the model.
+      kiva = false
+      surfaces.values.each do |surface|
+        next if kiva
+        kiva = true if surface.has_key?(:kiva)
+      end
+
+      if kiva
+        arg = "TotalExposedPerimeter"
+        foundation_kiva_settings = model.getFoundationKivaSettings
+        foundation_kiva_settings.setName("TBD-generated Kiva settings template")
```

Comment:
By explicitly declaring the Kiva foundation settings, it provides a placeholder or mechanism for users to (manually) tweak settings.